### ISSUE TITLE
Resume video file doesn't work, plays from beginning always

### ISSUE THREAD

**Issue by jmediamanager** [japanesemediamanager/MyAnime3#14]
**Reported by ignaciogarciaaguirre, Oct 22, 2013**

*What steps will reproduce the problem?*

1. Play video.
2. Stop video.
3. Replay video.

*What is the expected output? What do you see instead?*
Option to resume or start from the beginning.
Starts from the beginning always.

*What version of the product are you using? On what operating system?*
MA3 3.1.32.0
MediaPortal 1.5.0 Release
Titan Skin

```
00000005 - 22-10-2013 12:26:43 - ImageLoad: Finished
00000001 - 22-10-2013 12:26:43 - SetFacade List Mode: Episode
00000001 - 22-10-2013 12:26:43 - SetFacade: Filters: False - Groups: False - Series: False - Episodes: True
00000024 - 22-10-2013 12:26:43 - GOT FANART details in: 5.0003 ms (.hack//G.U. Returner)
00000024 - 22-10-2013 12:26:43 - LOADING FANART: .hack//G.U. Returner - C:\ProgramData\Team MediaPortal\MediaPortal\Thumbs\Anime3\TvDB\fanart\original\79099-10.jpg
00000024 - 22-10-2013 12:26:43 - Report ItemToAutoSelect: 0
00000024 - 22-10-2013 12:26:43 - SetFacade List Mode: Episode
00000024 - 22-10-2013 12:26:43 - SetFacade: Filters: False - Groups: False - Series: False - Episodes: True
00000001 - 22-10-2013 12:26:43 - [Anime2:\anime3_DVDDIVX.png]
00000001 - 22-10-2013 12:26:51 - Selected to play: 1 - OVA
00000001 - 22-10-2013 12:26:51 - Filetoplay: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:26:51 - Getting time stopped for : Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:26:51 - Time stopped for : Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - 0
00000001 - 22-10-2013 12:26:52 - PlayBackOpIsOfConcern: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - Video - MyAnimePlugin3.ViewModel.AnimeEpisodeVM
00000001 - 22-10-2013 12:26:52 - Playback started for: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:27:26 - OnPlayBackStopped: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - 33 - Video
00000001 - 22-10-2013 12:27:26 - PlayBackOpIsOfConcern: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - Video - MyAnimePlugin3.ViewModel.AnimeEpisodeVM
00000001 - 22-10-2013 12:27:26 - Playback stopped for: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:27:26 - Checking for set watched
00000001 - 22-10-2013 12:27:26 - Starting page load...
00000001 - 22-10-2013 12:27:26 - Adding hook to hook event handler
00000001 - 22-10-2013 12:27:26 - SetFacade List Mode: Episode
00000001 - 22-10-2013 12:27:26 - SetFacade: Filters: False - Groups: False - Series: False - Episodes: True
00000001 - 22-10-2013 12:27:26 - C:\ProgramData\Team MediaPortal\MediaPortal\Skin\Titan\Anime3_SkinSettings.xml
00000001 - 22-10-2013 12:27:26 - Loading Logos
00000001 - 22-10-2013 12:27:26 - Thumbs Setting Folder: C:\ProgramData\Team MediaPortal\MediaPortal\Thumbs
00000001 - 22-10-2013 12:27:26 - SetFacade List Mode: Episode
00000001 - 22-10-2013 12:27:26 - SetFacade: Filters: False - Groups: False - Series: False - Episodes: True
00000024 - 22-10-2013 12:27:26 - GOT FANART details in: 4.0002 ms (.hack//G.U. Returner)
00000024 - 22-10-2013 12:27:26 - LOADING FANART: .hack//G.U. Returner - C:\ProgramData\Team MediaPortal\MediaPortal\Thumbs\Anime3\TvDB\fanart\original\79099-5.jpg
00000024 - 22-10-2013 12:27:26 - Report ItemToAutoSelect: 0
00000024 - 22-10-2013 12:27:26 - SetFacade List Mode: Episode
00000024 - 22-10-2013 12:27:26 - SetFacade: Filters: False - Groups: False - Series: False - Episodes: True
00000001 - 22-10-2013 12:27:32 - Selected to play: 1 - OVA
00000001 - 22-10-2013 12:27:32 - Filetoplay: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:27:32 - Getting time stopped for : Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:27:32 - Time stopped for : Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - 0
00000001 - 22-10-2013 12:27:32 - PlayBackOpIsOfConcern: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - Video - MyAnimePlugin3.ViewModel.AnimeEpisodeVM
00000001 - 22-10-2013 12:27:32 - Playback started for: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
00000001 - 22-10-2013 12:27:36 - OnPlayBackStopped: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - 3 - Video
00000001 - 22-10-2013 12:27:36 - PlayBackOpIsOfConcern: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv - Video - MyAnimePlugin3.ViewModel.AnimeEpisodeVM
00000001 - 22-10-2013 12:27:36 - Playback stopped for: Y:\anime\.hackG.U. Returner\[LuPerry]_dot_hack_GU_-_returner[496A700E].mkv
```

**Comment by RickDB** [comment]
Tested here with latest MA3 version and getting the correct resume time and resumes like expected, so this one looks fixed.

**Comment by ElementalCrisis** [comment]
Fixed in 3.6.0.10.
